packages/bundle-source/src/index.js

```diff
@@ -192,6 +209,17 @@ async function bundleNestedEvaluateAndGetExports(
   });
   // console.log(output);
 
+  // Find the longest common prefix of all the source file names.
+  // We shorten the fileNames to make the bundle output deterministic.
+  const fileNameToUrlPath = fileName =>
+    readPowers.pathToFileURL(pathResolve(startFilename, fileName)).pathname;
+  const pathnames = output.map(({ fileName }) => fileNameToUrlPath(fileName));
+  const longestPrefix = longestCommonPrefix(pathnames);
+
+  // Ensure the prefix ends with a slash.
+  const pathnameEndPos = longestPrefix.lastIndexOf('/');
+  const pathnamePrefix = longestPrefix.slice(0, pathnameEndPos + 1);
```

Comment:
I reckon that this is correct even for pairs of paths like `common/common-left` and `common/common-right`. My intuition would have been to implement `longestCommonPrefix` as `longestCommonPath` such that it guarantees that the prefix occurs at a path delimiter, but this works too.

Also, doing the math in the URL domain would be more portable, since it normalizes delimiters.

Reply:
> Also, doing the math in the URL domain would be more portable, since it normalizes delimiters.

Not sure what you mean by this... the math is currently done on the normalised `url.pathname`.  Are you suggesting that something different would be better?

Reply:
No, I misread. You’re already doing what I suggested. Thanks!